src/console.c

```diff
@@ -46,7 +47,7 @@ static console_string_t *Con_AllocString(int32_t level, const char *string) {
 	str->size = strlen(str->chars);
 	str->length = StrColorLen(str->chars);
 
-	str->timestamp = Sys_Milliseconds();
+	str->timestamp = SDL_GetTicks();
 
```

Comment:
I wonder if this one could be `quetoo.time`, which is updated each frame (see `src/main/main.c`).